api/app/src/Migrations/Version282.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version282 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Adds index for created_by_id column on dd_user';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE dd_user DROP CONSTRAINT dd_user_created_by_fk');
+        $this->addSql('ALTER TABLE dd_user ADD CONSTRAINT FK_6764AB8BB03A8386 FOREIGN KEY (created_by_id) REFERENCES dd_user (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_6764AB8BB03A8386 ON dd_user (created_by_id)');
```

Comment:
If I'm understanding this correctly, I'm not sure we can do this because an admin user can and should be able to invite multiple users, so the value in the `created_by_id` column won't be unique.